azext_edge/tests/edge/checks/int/test_dataflow_int.py

```diff
@@ -26,7 +26,7 @@
 @pytest.mark.parametrize("detail_level", ResourceOutputDetailLevel.list())
 @pytest.mark.parametrize("resource_kind", DataflowResourceKinds.list() + [None])
 # TODO: figure out if name match should be a general test vs each service (minimize test runs)
-@pytest.mark.parametrize("resource_match", [None, "*dataflow-operator*", "aio-dataflow-operator*", generate_names()])
+@pytest.mark.parametrize("resource_match", [None, "aio-dataflow-operator*", generate_names()])
```

Comment:
tbh I am fine with removing resource match from here since it mostly increases number of runs